src/modules/multi-file-upload/services/multi.file.upload.service.js

```diff
@@ -142,11 +142,23 @@ const prepareStruct = ({ parentInfo, config, languageCode }, data) => {
         .then((response) => response.json())
         .catch(() => window.eZ.helpers.notification.showErrorNotification('Cannot get content type by identifier'))
         .then((response) => {
+            const fileValue = {
+                fileName: data.file.name,
+                data: data.fileReader.result.replace(/^.*;base64,/, ''),
+            }
+
+            if (data.file.type.startsWith('image/')) {
+                fileValue.alternativeText = data.file.name;
+            }
+
             const fields = [
-                { fieldDefinitionIdentifier: mapping.nameFieldIdentifier, fieldValue: data.file.name },
+                {
+                    fieldDefinitionIdentifier: mapping.nameFieldIdentifier,
+                    fieldValue: data.file.name
```

Comment:
If multiline, the last item should have `,` at the end.

I think it was good with a single line, especially with that the second item also could be a single line now.

Reply:
Reverted to single line in  https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/292/commits/a9149aee84ef4eaa608ed4d6d2f288af6c58ef7f